toolkit/imageconfigs/edge-image-dev.json

```diff
@@ -77,7 +77,8 @@
                 "packagelists/os-ab-update.json",
                 "packagelists/vpro-amt-packages.json",
                 "packagelists/docker.json",
-                "packagelists/k3s.json"
+                "packagelists/k3s.json",
+                "packagelists/helm.json"
```

Comment:
is this needed for edge-image too? 
https://github.com/open-edge-platform/edge-microvisor-toolkit/pull/258/files 
i see this applied for edge-image-desktop-virtualization